.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+  schedule:
+    - cron: '0 1 * * *' # Test every day at 1AM
+  
+jobs:
+  Full_Analysis:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Add conda to system path
+        run: |
+          echo $CONDA/bin >> $GITHUB_PATH 
+
+      - name: Install dependencies
+        run: |
+          conda env update --file environment.yml --name base
+
+      - name: Conda list
+        run: |
+          conda list
```

Comment:
Is this step so we have a history of what dependency versions were installed?

Reply:
None of the versions are pinned (intentionally) so we have to list if we want to know what versions were used for a specific run.

Reply:
Makes sense, just wanted to be sure.